data/ctf_platforms/attack_defense.json

```diff
@@ -45,6 +45,13 @@
       "language": "Python",
       "price": "Free"
     },
+    {
+      "name": "Parrot CTFs",
```

Comment:
Could you sort it alphabetically please?

Reply:
yes I can. I am sorry I was not online lemme fix all this now.